.travis.yml

```diff
@@ -16,6 +16,7 @@ python:
 install:
   - pip install -U pip wheel
   - pip install sphinx
+  - pip install coverage
```

Comment:
Please add the relevant commands to [install.sh](https://github.com/prasadtalasila/IRCLogParser/blob/dev/install.sh) as well so that we can run the coverage test on development machines as well.